src/Test/Spec/Reporter/TC.purs

```diff
@@ -0,0 +1,100 @@
+-- | Team City reporter, also the one used for intellij
+module Test.Spec.Reporter.TC (tcReporter, teamcity) where
```

Comment:
```suggestion
module Test.Spec.Reporter.TeamCity (teamcityReporter, teamcity) where
```

I'd much prefer if I could know or at least google what reporter this is by the filename alone. I don't think TeamCity is widespread enough for people to know that `TC` stands for it in the context of this library.